data/companies.yml

```diff
@@ -77,18 +77,12 @@ users:
   - name: "Autodesk"
     logo: "/logos/autodesk.png"
     url: "https://www.autodesk.com"
-  - name: "AutoTrader UK"
```

Comment:
Why is this removed? I see they are an active users of Isito.

Reply:
Because they're a case study, so they don't need to be in the list of "Also used by" logos below.

Reply:
fair enough.